README.md

```diff
@@ -16,7 +16,7 @@ a direct visit from the browser.
 
 ```javascript
 // Heroku defines the environment variable PORT, and requires the binding address to be 0.0.0.0
-var host = process.env.PORT ? '0.0.0.0' : '127.0.0.1';
+var host = process.env.HOST || '127.0.0.1';
```

Comment:
Okay, great. I've updated the PR according to your feedback. Should be fitting now, could you please check again?